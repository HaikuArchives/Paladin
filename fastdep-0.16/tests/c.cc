#define F

#if defined(A)
#include "a.h"
#elif defined(B)
#include "b.h"
#elif defined(F)
#include "f.h"
#endif

#if !defined(G)
#include "g.h"
#endif
