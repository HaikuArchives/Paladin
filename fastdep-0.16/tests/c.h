#ifndef C_H_
#define C_H_

#include "d.h"

#endif
