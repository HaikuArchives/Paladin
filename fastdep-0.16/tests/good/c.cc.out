c.o: c.cc \
	f.h \
	g.h
